backend/src/api/Add.py

```diff
@@ -0,0 +1,340 @@
+from src.utils.db_utils import get_db_connection
+from src.utils.db_handler import DBHandler
+from fastapi import APIRouter, HTTPException, UploadFile, Form, File
+from src.api.BaseModel import *
+from psycopg2 import errors
+from src.api.drive_utils import *
+
+conn = get_db_connection()
+
+db_handler = DBHandler(conn)
+
+router = APIRouter()
+
+@router.post("/addEdizioneCorso")
+def addEdizioneCorso(edizioneCorso: AddEdizioneCorso):
```

Comment:
Cosi va fatto dentro ogni chiamata API

```suggestion
def addEdizioneCorso(
    edizioneCorso: AddEdizioneCorso,
    db_handler: DBHandler = Depends(get_db_handler)
):
```